src/components/Error/Oops.js

```diff
@@ -1,14 +1,20 @@
 import React, { Component } from 'react';
 import { Navbar, NavbarBrand, Col, Row, Button } from 'reactstrap';
+import NavFooter from '../Nav/NavFooter';
 
 class Oops extends Component {
     render() {
         return (
             <article className="container-fluid">
-                <Navbar className="container-fluid">
-                    <NavbarBrand>
-                        <img src="img/logo.png" alt="Kidney Precision Medicine Project Data Lake Uploader" className="logo" />
-                    </NavbarBrand>
+                <Navbar id="navbar" className="px-1 py-1 fixed-top">

```

Comment:
Don't we have a navbar object somewhere that we can use, rather than recreating it here?

Reply:
That was my first try.  It resulted in a lot of errors, because we need to inject state from redux.  Because we're redoing this header as part of the navigational changes, copying the relevant code rather than refactoring the code we will make obsolete seemed the more expedient solution with the least throwaway work.